test/n2z/utility.py

```diff
@@ -1,20 +1,64 @@
 """Simple utility functions used in the NetCDF-to-Zarr test notebook."""
 
+import boto3
+from filecmp import dircmp
+import os
+from typing import List
+
 
 def print_success(success_string: str) -> str:
     """Print a success message, with formatting for green text."""
     print(f'\033[92mSuccess: {success_string}\033[0m')
 
 
+def get_zarr_stores(results: dict) -> List[dict]:
+    """Return all results items that are type "application/x-zarr"."""
+    return [link for
```

Comment:
_Super_ nit: I think the style convention for breaking a line in a list comprehension is to break before the `for`, `if`, etc. I can't find an exact reference, and `pylint` didn't complain, but it definitely seems to be the norm:

```
return [link
        for link in results.get('links')
        if link.get('type') == 'application/x-zarr']
```